website/www/site/content/en/get-started/downloads.md

```diff
@@ -93,39 +93,43 @@ versions denoted `0.x.y`.
 
 ## Releases
 
-### 2.67.0 (2025-08-12)
+### Current release
 
-Official [source code download](https://archive.apache.org/dist/beam/2.67.0/apache-beam-2.67.0-source-release.zip).
-[SHA-512](https://archive.apache.org/dist/beam/2.67.0/apache-beam-2.67.0-source-release.zip.sha512).
-[signature](https://archive.apache.org/dist/beam/2.67.0/apache-beam-2.67.0-source-release.zip.asc).
+#### 2.67.0 (2025-08-12)
+
+Official [source code download](https://www.apache.org/dyn/closer.lua/beam/2.67.0/apache-beam-2.67.0-source-release.zip).
```

Comment:
Should this be https://dlcdn.apache.org/beam/2.67.0/apache-beam-2.67.0-source-release.zip? If I go to the link you've included, it suggests redirecting to that.